### PR TITLE
Singly linked list using unique key code in cpp

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Singly linked list using unique key code in cpp  #3393

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
added SLL Operation Using Unique Key in c-plus-plus/ds #3393 and updated readme.md

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 
- Add a beginner-friendly description about your Code at the top
- Add description of functions 
- Add sample I/O at the end of the Code
- Add complexity at the end of the code
- Use a Code formatter to format the Code

**Comment by anshmishra010** [comment]
- [x] Added a beginner-friendly description about your Code at the top
- [x] Added description of functions
- [x] Added sample I/O at the end of the Code
- [x] Added complexity at the end of the code
- [x] Formatted the Code

**Comment by anshmishra010** [comment]
Hello harsh ,
I have updated the code , please check and reply .

**Review by anushkrishnav — APPROVED** [review]

File: C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp [hunk 1]
```diff
@@ -0,0 +1,314 @@
+/*
+This is singly linked list implementation in which a unique key is alloted while entering the new node to the linked list, and you cannot give same key to the different node. In this code you can give duplicate data still it work fine.
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp]: Modularize this and make the lines shorter and more readable

File: C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+input: 1 1 100 1 2 101 4 2 6
+outout: (1,100)--> NULL
+
+*/
+
+/*
+Time complexity - O(n)
+
+Space complexity - O(n)
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
@anshmishra010 Kindly do the changes above mentioned.

File: C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp (hunk 1 above)
Comment (anshmishra010) [reply C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp]: - [x] Modularized the description and made the lines shorter and more readable.

File: C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp (hunk 2 above)
Comment (anshmishra010) [reply C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp]: Added extra lines at the EOF

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**